classes/submission/Collector.php

```diff
@@ -379,41 +387,57 @@ public function getQueryBuilder(): Builder
-                );
-            // Only get overdue assignments on active review rounds
-            $q->whereNotIn('rr.status', [
-                ReviewRound::REVIEW_ROUND_STATUS_RESUBMIT_FOR_REVIEW,
-                ReviewRound::REVIEW_ROUND_STATUS_SENT_TO_EXTERNAL,
-                ReviewRound::REVIEW_ROUND_STATUS_ACCEPTED,
-                ReviewRound::REVIEW_ROUND_STATUS_DECLINED,
-            ]);
-            $q->where(fn (Builder $q) =>
-                $q->where('raod.declined', '<>', 1)
-                    ->where('raod.cancelled', '<>', 1)
-                    ->where(fn (Builder $q) =>
-                        $q->where('raod.date_due', '<', Core::getCurrentDate(strtotime('tomorrow')))
-                            ->whereNull('raod.date_completed')
+            $q->whereIn(
+                's.submission_id',
+                fn (Builder $q) =>
+                $q->select('s.submission_id')
+                    ->from('submissions AS s')
+                    ->leftJoin('review_assignments as raod', 'raod.submission_id', '=', 's.submission_id')
+                    ->leftJoin(
+                        'review_rounds as rr',
+                        fn (Builder $table) =>
+                        $table->on('rr.submission_id', '=', 's.submission_id')
+                            ->on('raod.review_round_id', '=', 'rr.review_round_id')
                     )
-                    ->orWhere(fn (Builder $q) =>
-                        $q->where('raod.date_response_due', '<', Core::getCurrentDate(strtotime('tomorrow')))
-                        ->whereNull('raod.date_confirmed')
+                    ->whereNotIn('rr.status', [
+                        ReviewRound::REVIEW_ROUND_STATUS_RESUBMIT_FOR_REVIEW,
+                        ReviewRound::REVIEW_ROUND_STATUS_SENT_TO_EXTERNAL,
+                        ReviewRound::REVIEW_ROUND_STATUS_ACCEPTED,
+                        ReviewRound::REVIEW_ROUND_STATUS_DECLINED,
+                    ])
+                    ->where(
+                        fn (Builder $q) =>
+                                    $q->where('raod.declined', '<>', 1)
+                                        ->where('raod.cancelled', '<>', 1)
+                                        ->where(
+                                            fn (Builder $q) =>
+                                            $q->where('raod.date_due', '<', Core::getCurrentDate(strtotime('tomorrow')))
+                                                ->whereNull('raod.date_completed')
+                                        )
+                                        ->orWhere(
+                                            fn (Builder $q) =>
+                                            $q->where('raod.date_response_due', '<', Core::getCurrentDate(strtotime('tomorrow')))
+                                                ->whereNull('raod.date_confirmed')
+                                        )
                     )
             );
         }
```

Comment:
The formatter made quite a few changes to the file, but these are the lines specific to the issue at hand